app/src/sharedTest/java/org/oppia/app/testing/InputInteractionViewTestActivityTest.kt

```diff
@@ -288,9 +288,64 @@ class InputInteractionViewTestActivityTest {
   fun testFractionInputInteractionView_withInputtedLongNumber_clickSubmitButton_numberTooLongErrorIsDisplayed() {
     ActivityScenario.launch(InputInteractionViewTestActivity::class.java).use {
       onView(withId(R.id.test_fraction_input_interaction_view)).perform(typeText("12345678"))
-      closeSoftKeyboard ()
+      closeSoftKeyboard()
       onView(withId(R.id.submit_button)).check(matches(isDisplayed())).perform(click())
       onView(withId(R.id.fraction_input_error)).check(matches(withText(R.string.fraction_error_larger_than_seven_digits)))
     }
   }
+
+  @Test
+  fun testNumericInputInteractionView_withInputtedInvalidCharacter_invalidCharacterErrorIsDisplayed() {
```

Comment:
Did you test the case where the number is successfully parsed? If not, please add. Also, it might be nice to ones for decimals & negatives, as well (e.g. ~3 success test cases).

Reply:
It was already there. added new test for negative number.